cmd/kubelet/app/options/options.go

```diff
@@ -465,7 +465,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.BoolVar(&c.RunOnce, "runonce", c.RunOnce, "If true, exit after spawning pods from static pod files or remote urls. Exclusive with --enable-server")
 
 	fs.BoolVar(&c.CPUCFSQuota, "cpu-cfs-quota", c.CPUCFSQuota, "Enable CPU CFS quota enforcement for containers that specify CPU limits")
-	fs.DurationVar(&c.CPUCFSQuotaPeriod.Duration, "cpu-cfs-quota-period", c.CPUCFSQuotaPeriod.Duration, "Sets CPU CFS quota period value, cpu.cfs_period_us, defaults to Linux Kernel default")
+	fs.DurationVar(&c.CPUCFSQuotaPeriod.Duration, "cpu-cfs-quota-period", c.CPUCFSQuotaPeriod.Duration, "Sets CPU CFS quota period value, cpu.cfs_period_us, defaults to Linux Kernel default (100usec)")
```

Comment:
this makes a super confusing message:

> Sets CPU CFS quota period value, cpu.cfs_period_us, defaults to Linux Kernel default (100μs) (default 100ms)

Reply:
I'm also not sure this change is correct... this is a time.Duration, and is currently defaulted to 100 milliseconds:

```go
	if obj.CPUCFSQuotaPeriod == nil {
		obj.CPUCFSQuotaPeriod = &metav1.Duration{Duration: 100 * time.Millisecond}
	}
```

Reply:
It is supposed to be 100 microseconds to be in sync with Linux Kernel. Thanks a lot for noticing it! I've changed the line you mentioned.

As far as I understand, you need to enable the flag `CustomCPUCFSQuotaPeriod` to use that value, and if you set a non-default `cpuCFSQuotaPeriod`, nothing changes for you.
However, if you have `CustomCPUCFSQuotaPeriod` enabled but haven't set `cpuCFSQuotaPeriod`, after my change, it will become in sync with Linux Kernel default 1000x smaller, 100µs instead of 100ms. That is desirable behaviour, but it's magnitudes different from just the documentation change I was thinking about initially.

Reply:
@liggitt I think in my pr I assumed that all the code comments were correct and these were 100ms (we even had the production problem with not being able to have lower latency than 100ms), so I set the default to 100ms. However it's only used if the featureGate is enabled, and no value is set. I think it's rarely a combination setting that anyone would do.
Of course we can't be 100% sure, but tbh maybe it's a bug fix, that no one before observed.
@derekwaynecarr and @dims wdyt from sig-node point of view ?

Reply:
> However it's only used if the featureGate is enabled, and no value is set. I think it's rarely a combination setting that anyone would do.

are all uses of the kubelet config value gated by the feature gate? if so, the fact that that is still an alpha feature gate makes it more acceptable to change this default, imo

Reply:
yes all uses are gated by the feature gate